package.json

```diff
@@ -104,6 +104,9 @@
     "ts-jest": "^27.1.4",
     "typescript": "4.6.3"
   },
+  "overrides": {
+    "@cucumber/messages": "^18.0.0"
+  },
```

Comment:
Why is this required?
Gherkin already comes with mesages 18.0.0. Is this to make it explicit?

Reply:
See the comment above - fake-cucumber is bringing an older version along so this is to neutralise that.

Reply:
Thanks for the explanation

Good idea to build fake-cucumber as a standalone 👍